components/resolved_ts/src/advance.rs

```diff
@@ -117,7 +117,7 @@ impl AdvanceTsWorker {
             // optimizations like async commit is enabled.
             // Note: This step must be done before scheduling `Task::MinTs` task, and the
             // resolver must be checked in or after `Task::MinTs`' execution.
-            cm.update_max_ts(min_ts);
+            cm.update_max_ts(min_ts, "resolved-ts").unwrap();
```

Comment:
Following the discussion above, if it falls back to approximate checks and a misjudgment occurs, could TiKV still panic?
This could be a risk picking to the release versions.

Reply:
I made a change here: when update_max_ts fails, just skip this attempt instead of panicking.